tests/mock/clitools.rs

```diff
@@ -1107,19 +1107,14 @@ fn mock_bin(name: &str, version: &str, version_hash: &str) -> Vec<MockFile> {
             let status = Command::new("rustc")
                 .arg(&source_path)
                 .arg("-C").arg("panic=abort")
+                .arg("-C").arg("debuginfo=0")
```

Comment:
One reason we stripped was because it used a lot of disk space during tests otherwise.  Is this sufficiently close to a stripped binary size so as to be equivalent in terms of disk space savings?  The test suite deletes things as it goes these days so it's less of an issue if not, but it could still present an issue for some if it's not close to as small.

Reply:
I expect that the binary passing `debuginfo=0` is equivalent to the stripped binary,
but they are not.
I asked on discord. Hope the developers give use an explaination.

Reply:
The `debuginfo=0` is the default option when not specified.
We still need to strip the output binary because it contains
the debug table from std/core.